server/config/client.go

```diff
@@ -227,6 +227,12 @@ func GenerateClientConfig(c *model.Config, telemetryID string, license *model.Li
 			props["PostAcknowledgements"] = "true"
 			props["ScheduledPosts"] = strconv.FormatBool(*c.ServiceSettings.ScheduledPosts)
 		}
+
+		if license.SkuShortName == model.LicenseShortSkuEnterprise {
+			props["MobileEnableBiometrics"] = strconv.FormatBool(*c.NativeAppSettings.MobileEnableBiometrics)
+			props["MobilePreventScreenCapture"] = strconv.FormatBool(*c.NativeAppSettings.MobilePreventScreenCapture)
+			props["MobileJailbreakProtection"] = strconv.FormatBool(*c.NativeAppSettings.MobileJailbreakProtection)
+		}
```

Comment:
You may need to update the branch as this change has been introduced here: https://github.com/mattermost/mattermost/pull/30411